kdb/key.go

```diff
@@ -228,6 +237,7 @@ func (k *ckey) String() string {
 	name := k.Name()
 	value := k.Value()
 
+	// TODO REVIEW: Why is this needed?
```

Comment:
To make it obvious that the value is empty - the output of this function is nothing a user should see, it's more for development purposes

Reply:
I renamed the Value() function to String() and removed the previous String() function